폰트 적용 완료

PSO01 opened the pull request.

PSO01 committed:
[FEAT]웹/모바일 뷰 일관성 유지 AppWrapper.jsx생성, overflow:hidden삭제, App.jsx 수정

leeeeejieun commented on client/src/App.jsx:
AppWrapper 컴포넌트 import 해야 합니다
추가로 폰트 설정까지 부탁드릴게요

PSO01 committed:
[FEAT] 폰트 설정, AppWrapper 컴포넌트 import

PSO01 committed:
[FIX] 폰트 url 수정

leeeeejieun commented on client/src/styles/GlobalStyle.jsx:
```diff
@@ -15,7 +15,6 @@ const GlobalStyle = createGlobalStyle`
     width: 100%;
     height: 100vh; 
-    overflow: hidden;
     font-family: 'Gaegu', sans-serif;
```
현재 추가한 폰트가 웹 페이지에 전체적으로 사용되는 폰트라면 body의 font-failly를 해당 폰트로 변경해주세요

leeeeejieun commented on client/src/styles/GlobalStyle.jsx:
```diff
@@ -40,6 +39,11 @@ const GlobalStyle = createGlobalStyle`
    }
+
+   @font-face {
+    font-family:'Hakgyoansim Geurimilgi.ttf'; 
```
폰트명이 너무 길면 나중에 사용하실 때 불편하실 듯 합니다 적절한 폰트명으로 바꿔주시고 확장자도 지워주세요

PSO01 committed:
[RENAME] 폰트 이름 수정

PSO01 committed:
[FIX] font-family를 현재폰트로 변경

PSO01 committed:
[FEAT] 새 폰트 추가

leeeeejieun committed:
[FIX] 폰트 오류 수정

- fonts 폴더의 위치를 public 폴더의 하위 폴더로 변경
- 각 폰트마다 @font-face 블록 적용
- 폰트 파일명을 영어로 수정
